src/exportImage.py

```diff
@@ -59,10 +61,17 @@ def export(self, bbox, filename, url, taskid, directoryName, max_zoom):
                 result = self.create_geopackage(
                     bbox, filename, url, taskid, full_path, resolution)
 
-                if result is not None:
+                if result:
                     self.create_index(filename, full_path)
+                    if (self.__config["storage_provider"] == StorageProvider.S3.value):
+                        self.upload_to_s3(filename, directoryName,
+                                          output_format, full_path)
+
```

Comment:
after upload do:
1. get full size // file_size = path.getsize(fullPath)
2.  self.delete_local_directory(directoryName)
3. send to save_update the full_size as an optional parametrer